app/src/main/java/ch/epfl/sdp/blindwar/ui/StatisticsActivity.kt

```diff
@@ -1,28 +1,77 @@
 package ch.epfl.sdp.blindwar.ui
 
+import android.content.ContentValues
 import android.os.Bundle
+import android.util.Log
 import android.view.View
-import android.widget.AdapterView
-import android.widget.ArrayAdapter
-import android.widget.Spinner
-import android.widget.Toast
+import android.widget.*
 import androidx.appcompat.app.AppCompatActivity
 import ch.epfl.sdp.blindwar.R
+import ch.epfl.sdp.blindwar.user.AppStatistics
+import ch.epfl.sdp.blindwar.user.Mode
+import ch.epfl.sdp.blindwar.user.Result
+import ch.epfl.sdp.blindwar.user.User
+import com.google.firebase.database.*
+import com.google.firebase.database.ktx.getValue
 
 
 class StatisticsActivity : AppCompatActivity() {
 
-    //Float values for stats (only for demo)
-    private var wins = 45
-    private var losses = 35
-    private var draws = 100 - wins - losses
+    val firebaseDatabase: FirebaseDatabase = FirebaseDatabase.getInstance()
+    val databaseReference = firebaseDatabase.getReference("Data")
+
+    var userStatistics: AppStatistics = AppStatistics()
```

Comment:
Maybe decouple the user's info from the Activity to separate the concerns and to make it more accessible for other parts of the app